gen/README.md

```diff
@@ -4,14 +4,16 @@ GDAL.jl generation documentation
 ### Install Clang.jl
 Because a few tricks are used to be able to properly write documentation,
 this fork of Clang.jl is used: https://github.com/visr/Clang.jl (branch gdal)
+For the generation process `julia 0.5-dev` based on `LLVM 3.7.1` and with
+`BUILD_LLVM_CLANG=1` in `Make.user` is used.
 
 ### Make sure you have the GDAL header files
-The `wrap_gdal.jl` script looks in `/usr/local/include` by default.
+The `wrap_gdal.jl` script finds them in the directory set in the `gdalpath` variable.
 
 ### Prepare the GDAL Doxygen XML
 1. In the GDAL SVN, run Doxygen with `GENERATE_XML = YES`
-2. In the `xml` output dir, run `combine.xslt` to combine all documentation in one XML
-3. Copy this file to `GDAL.jl/gen/all.xml`
+2. Follow the instructions in `combine_gdal_doxygen_xml.py`
+3. Copy the result to `GDAL.jl/gen/doxygen.xml`
```

Comment:
Here is the version I am using:
https://s3.eu-central-1.amazonaws.com/visr/julia/gdal/doxygen.xml
I'll add this link to the instructions as well. But cool that you are trying it yourself as well. From the error it seems the XML you are using is not well-formed (mismatched tag), did you check this?